main.py

```diff
@@ -1,7 +1,49 @@
-import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
 
+import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.bulk_create([
```

Comment:
follow the checklist p2 and p3: prepare data in tuples and then create entries to db in a cycle